### PR TITLE
Updated license notices for javascript menus.

### PR DESCRIPTION
**Pull request by aashaikh**
Added full Apache license notice for modifications to original MIT-licensed files.

### CODE REVIEW

File: openconfig_pyang/plugins/templates/js/yangdoc.menu.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
+// Copyright 2015 Google, Inc.
```
Comment (robshakir) [inline openconfig_pyang/plugins/templates/js/yangdoc.menu.js]: `Modifications copyright...` ?

File: openconfig_pyang/plugins/templates/css/yangdoc.menu.css [hunk 2]
```diff
@@ -1,4 +1,18 @@
 /*
+* Copyright 2015 Google, Inc.
```
Comment (robshakir) [inline openconfig_pyang/plugins/templates/css/yangdoc.menu.css]: `Modifications` again :sake:

**Review by robshakir — CHANGES_REQUESTED** [review]
Please sync to master and ensure that CI passes :-)

**Comment by aashaikh** [comment]
thanks for the review ! -- updated per your suggestions -- PTAL.

**Review by robshakir — APPROVED** [review]

**Pull request merged**